ITEM_en.tsv

```diff
@@ -1157,7 +1157,7 @@ ITEM_20150317_001250	Tomb Leather Gloves
 ITEM_20150317_001251	Tomb Gauntlets
 ITEM_20150317_001252	Walker
 ITEM_20150317_001253	For Mine Armor
-ITEM_20150317_001254	Crafted by a recipe resulting from hundreds of years' worth of miners' experiences and wisdom.
+ITEM_20150317_001254	Crafted by a recipe resulting from hundreds of years worth of miners experience and wisdom.
```

Comment:
" years " -> " years' "
" miners " -> " miners' "

Since both nouns are plural and possessive, the apostrophe afterwards is appropriate. As odd as it is, this applies to the phrase "years' worth" as well.

e.g. - " three days' journey = journey of three days"